### PR TITLE
We are sometimes hiring

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Reinstate "we are hiring" messages in:

- frontend repo readme
- browser console panel
- HTML source

(originally removed in #16641 and #16484)

Messages in console panel and HTML source are now behind a feature switch for easy on/off-ability 

## What is the value of this and can you measure success?

Free recruitment advertising

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Screenshots

**Browser console**

picture 334

**HTML source**

picture 335

## Tested in CODE?

No

### CODE REVIEW

File: common/app/views/fragments/head.scala.html [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+     __        __                      _     _      _
+     \ \      / /__    __ _ _ __ ___  | |__ (_)_ __(_)_ __   __ _
+      \ \ /\ / / _ \  / _` | '__/ _ \ | '_ \| | '__| | '_ \ / _` |
+       \ V  V /  __/ | (_| | | |  __/ | | | | | |  | | | | | (_| |
+        \_/\_/ \___|  \__,_|_|  \___| |_| |_|_|_|  |_|_| |_|\__, |
+                                                            |___/
+    Ever thought about joining us?
+    https://workforus.theguardian.com/careers/digital-development/
+     --->
+}
```
Comment (SiAdcock) [inline common/app/views/fragments/head.scala.html]: Legacy support

File: static/src/javascripts/bootstraps/standard/main.js [hunk 2]
```diff
@@ -47,8 +47,29 @@ const setAdTestCookie = (): void => {
     }
 };
 
+const showHiringMessage = (): void => {
+    try {
+        if (config.get('switches.weAreHiring')) {
```
Comment (SiAdcock) [inline static/src/javascripts/bootstraps/standard/main.js]: Maybe we should disable in dev to reduce console clutter

Comment (gustavpursche) [reply static/src/javascripts/bootstraps/standard/main.js]: I'd prefer that 👍

File: common/app/conf/switches/FeatureSwitches.scala [hunk 3]
```diff
@@ -525,4 +525,14 @@ trait FeatureSwitches {
     sellByDate = new LocalDate(2018, 10, 24),
     exposeClientSide = false
   )
+
+  val WeAreHiring = Switch(
```
Comment (NataliaLKB) [inline common/app/conf/switches/FeatureSwitches.scala]: Awesome idea to make this a switch 💯

File: common/app/conf/switches/FeatureSwitches.scala [hunk 4]
```diff
@@ -525,4 +525,14 @@ trait FeatureSwitches {
...
+    SwitchGroup.Feature,
+    "we-are-hiring",
+    "When ON, hiring messages will appear in browser console and HTML source",
+    owners = Seq(Owner.withGithub("siadcock")),
```
Comment (NataliaLKB) [inline common/app/conf/switches/FeatureSwitches.scala]: Should we make the team an owner as well?

**Review by NataliaLKB — APPROVED** [review]
looks good!

File: common/app/conf/switches/FeatureSwitches.scala [hunk 5]
```diff
@@ -530,7 +530,7 @@ trait FeatureSwitches {
     SwitchGroup.Feature,
     "we-are-hiring",
     "When ON, hiring messages will appear in browser console and HTML source",
-    owners = Seq(Owner.withGithub("siadcock")),
+    owners = Seq(Owner.withName("dotcom/platform")),
```
Comment (NataliaLKB) [inline common/app/conf/switches/FeatureSwitches.scala]: I think in the past we have used `dotcom.platform` https://github.com/guardian/frontend/blob/f347296b927fc0ee74ff4413b4cbb9f4e948bf51/common/app/mvt/MultiVariateTesting.scala#L94

**Pull request merged**